src/keyboard.js

```diff
@@ -0,0 +1,10 @@
+$(function() {
+  var $write = $('#write'),
+
+    $(`.keyboard-row li`).click(function() {
+      var $this = $(this),
```

Comment:
Right now I am getting a build error because of a syntax error here. Notice that we are running a statement here not adding properties to an object. So this comma should be a `;`.